README.md

```diff
@@ -96,6 +96,36 @@ UI.
 Conduit can be configured through command line parameters. To view the full list
 of available options, run `./conduit --help`.
 
+### Homebrew
+
+Make sure you have [homebrew](https://brew.sh/) installed on your machine, then run:
+
+```sh
+brew install conduit
+brew upgrade conduit
```

Comment:
No need to upgrade Conduit, install should be enough. If anything, I'd say to run `brew update` before installing to make sure the latest homebrew formula is used.